client/web/compose/src/components/Common/RecordToolbar.vue

```diff
@@ -19,10 +20,16 @@
           @click.prevent="$emit('back')"
         >
           <font-awesome-icon
+            v-if="!showRecordModal"
             :icon="['fas', 'chevron-left']"
             class="back-icon"
           />
-          {{ labels.back || $t('label.back') }}
+          <font-awesome-icon
+            v-else
+            :icon="['fa', 'times']"
```

Comment:
Instead of repeating the whole font-awesome element, just have a computed for the icon name, since otherwise the two font awesome elements are the same

`['fas', backIcon]`